client/src/components/MapGeocoderWrapper.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div v-show='visible' class='geocoder-content-expanded'>
+    <div class='geocoder' id='geocoder'></div>
+    <div class='search-button' @click='this.visible = !this.visible'>
+      <img src='@/assets/icons/search.svg' />
```

Comment:
What's the source of this asset? Is it freely licensed?

Reply:
I'm a little concerned about using something that we don't know the license of, even temporarily. Can you message the author of the mocks to see if we can use it? Else I'm sure there's a freely licensed icon we can use.

Reply:
Reached out to author (Eric Schwartz) and he didn't know. Reaching out to Jackson and a few others and will resolve when I know for sure.

Reply:
Raanan confirmed that all icons in figma design system should be free to use.